include/unifex/await_transform.hpp

```diff
@@ -37,17 +37,22 @@
 
 namespace unifex {
 
-namespace _await_transform {
+namespace _await_tfx {
 
 template <typename Promise, typename Sender>
-class _as_awaitable {
+struct _awaitable {
```

Comment:
I hadn't added ADL-isolation for `_as_awaitable` because I don't think it's used in any ADL-calls.
It's only used by the `operator co_await` mechanics of the compiler.

I guess it doesn't hurt to do this though, just in case...

Reply:
Doesn't the compiler try to find `operator co_await` with ADL?

Reply:
Yes, I think you're right. It will attempt to find ADL-based overloads `operator co_await(_awaitable<Promise, Sender>)`.